app/services/experiment_control_service.rb

```diff
@@ -11,7 +11,12 @@ def start_current_patient_experiment(name, days_til_start, days_til_end, percent
 
       experiment.update!(state: "selecting", start_date: experiment_start.to_date, end_date: experiment_end.to_date)
 
-      eligible_ids = current_patient_candidates(experiment_start, experiment_end).shuffle!
+      eligible_ids = if name == "production test"
```

Comment:
`production test` feels like it should be a constant.

Reply:
Fair. I was hoping this would only be temporary code, but we could leave it in as a long term testing option.